components/ai_chat/core/browser/conversation_handler.cc

```diff
@@ -33,6 +36,10 @@
 #include "services/network/public/cpp/shared_url_loader_factory.h"
 #include "ui/base/l10n/l10n_util.h"
 
+#define STARTER_PROMPT(TYPE)                                         \
+  l10n_util::GetStringUTF8(IDS_AI_CHAT_STATIC_STARTER_TITLE_##TYPE), \
+      l10n_util::GetStringUTF8(IDS_AI_CHAT_STATIC_STARTER_PROMPT_##TYPE)
```

Comment:
pretty clever!

Reply:
@fallaciousreasoning In general macros should be avoided though https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros